nemo/collections/llm/gpt/model/megatron/hyena/hyena_utils.py

```diff
@@ -39,6 +39,11 @@
 
 from nemo.collections.llm.gpt.model.megatron.hyena.hyena_config import HyenaConfig
 
+with_state_p = True
+with_state_s = True
+with_state_m = True
+with_state_l = True
```

Comment:
As far as I know this was a temporary thing, we should only use state if inference_context is provided, right?